static/src/javascripts/projects/common/modules/navigation/new-header.js

```diff
@@ -188,22 +188,60 @@ const toggleMenu = (): void => {
     fastdom.write(update);
 };
 
+const toggleEditionPickerDropdown = () => {
+    const openClass = 'dropdown-menu--open';
+
+    fastdom.read(() => {
+        const editionPickerMenu = document.querySelector(
+            '.js-edition-dropdown-menu'
+        );
+        const trigger = document.querySelector('.js-edition-picker-trigger');
+
+        if (editionPickerMenu) {
+            const isOpen = editionPickerMenu.classList.contains(openClass);
+            const expandedAttr = isOpen ? 'false' : 'true';
+            const hiddenAttr = isOpen ? 'true' : 'false';
+
+            const toggleDropdown = () => {
+                if (trigger) {
+                    trigger.setAttribute('aria-expanded', expandedAttr);
+                }
+
+                editionPickerMenu.setAttribute('aria-hidden', hiddenAttr);
+                editionPickerMenu.classList.toggle(openClass, !isOpen);
+            };
+
+            fastdom.write(toggleDropdown);
+        }
```

Comment:
This can be written better, if you use `fastdom-promise` instead:

```js
fastdom.read(() => ({
  menu: document.querySelector('.js-edition-dropdown-menu'),
  trigger: document.querySelector('.js-edition-picker-trigger'),
})).then(els => {
  /* assigning logic */
  return fastdom.write(() => {
     /* writes */
  });
})
```